python/pyspark/sql/connect/session.py

```diff
@@ -413,11 +416,14 @@ def _inferSchemaFromList(
 
     def createDataFrame(
         self,
-        data: Union["pd.DataFrame", "np.ndarray", Iterable[Any]],
+        data: Union["pd.DataFrame", "np.ndarray", "pa.Table", Iterable[Any]],
         schema: Optional[Union[AtomicType, StructType, str, List[str], Tuple[str, ...]]] = None,
         samplingRatio: Optional[float] = None,
         verifySchema: Optional[bool] = None,
     ) -> "ParentDataFrame":
+        import pandas as pd
+        import pyarrow as pa
+
```

Comment:
```suggestion
```

Those packages are already imported top-level in this file as well?

Reply:
Thanks. Removed in 73a7886.